### ISSUE TITLE
Framework I

### ISSUE THREAD

**Issue by krottmas** [glossIT/transformations#3]
- [ ] Change folio number if needed
- [ ] AB to glosses and vice versa

**Comment by BernBa** [comment]
- divs for the chapters - how do we deal with marginal glosses in this case?

**Comment by krottmas** [comment]
> * divs for the chapters - how do we deal with marginal glosses in this case?

Was meinst Du damit ?? kannst Du ein Bild anhängen vom Use case? @BernBa

**Comment by krottmas** [comment]
Welchen Unterschied würde es machen wenn wir LB vor den jeweiligen Elementen ansiedeln?
momentan kann ich mit den LBs keine Textfelder basteln weil er die Leerstelle vor dem LB nimmt und nicht den eigentlichen Text.. Ich kann auch einfach zu doof sein dafür ^^

**Comment by BernBa** [comment]
Wir brauchen auch die Möglichkeit Glossen aus einer Region in eine andere zu verschieben. Ich hab das jetzt gerade wieder gemerkt, dass das öfters nicht passt.

**Comment by krottmas** [comment]
Könnte mir vorstellen, dass das recht einfach geht .. ich müsste nur hinweise hinbauen, dass sie den Kram nicht wo falsch hinschieben

**Comment by BernBa** [comment]
Ja, das ist natürlich leider immer möglich, aber dieses risiko müssen wir eingehen.

**Comment by krottmas** [comment]
Bis Dato können wir im ersten framework

- Textzeilen zu Glossen umwandeln
- Glossen zu Textzeigen umwandeln
- Metadaten einfügen
- ein zusätzliches locus feld einfügen, wenn notwendig
- den Fortschritt (not yet started, in progress, done) pro Bereich 
- glossen miteinander verbinden 


An einer Möglichkeit das ganze nach Validierung automatisch für den nächsten Schritt vorzubereiten arbeite ich gerade .. also ein basic xslt dass die zusätzlichen Gloss rausnimmt .. und das ganze in einen anderen ordner speichert für die weiterverarbeitung.. den zweiten schwung XSLTs würde ich eher gezielt nochmal extra ansteuern... damit wir da über die Einzelschritte etwas mehr überblick haben :) 

Fällt Dir noch was ein, was wir im ersten Framework brauchen könnten @BernBa?

**Comment by krottmas** [comment]
2 weitere "Funktionen"

- Glossen/Textzeilen verschieben
- Wenn alle Bereiche fertig sind, das File transformieren, um die zusätzlichen "gloss" zu entfernen
